fastlane/lib/fastlane/actions/xcodebuild.rb

```diff
@@ -248,6 +255,7 @@ def self.run(params)
 
         xcpretty_command = ""
         xcpretty_command = "| xcpretty #{xcpretty_args}" unless raw_buildlog
+        xcpretty_command = "#{xcpretty_command} --utf" unless raw_buildlog || !xcpretty_utf
```

Comment:
unless in combination with || and ! is pretty confusing, can you rewrite that line so it's easier to read?